tools/accuracy_checker/accuracy_checker/adapters/nlp.py

```diff
@@ -196,7 +196,34 @@ def process(self, raw, identifiers=None, frame_meta=None):
 
         return result
 
+class QuestionAnsweringBiDAFAdapter(Adapter):
+    __provider__ = 'bidaf_question_answering'
+    prediction_types = (QuestionAnsweringPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'start_pos': StringField(description="Output layer name for answer start position."),
```

Comment:
please add out suffix for emphasis that you expect output layer name, not an index